### ISSUE TITLE
Anonymous support of Steam Workshop

### ISSUE THREAD

**Issue by basxto** [Anuken/Mindustry-Suggestions#4989]
### Describe the content or mechanics you are proposing.

I don’t know if it’s possible, but might be worth checking if this can be enabled. SteamCMD allows to login anonymously and it can also download workshop items, but combining the both for mindustry so far fails.

```
$ mkdir /tmp/steamcmdtest/
$ steamcmd
Steam>force_install_dir  /tmp/steamcmdtest/
Steam>login anonymous
Steam>workshop_download_item 1127400 3030631911
Steam>quit
```

or short
```
mkdir /tmp/steamcmdtest/ && steamcmd +force_install_dir  /tmp/steamcmdtest/ +login anonymous +workshop_download_item 1127400 3030631911 +quit
```

Currently that throws:
```
Downloading item 3030631911 ...
ERROR! Download item 3030631911 failed (Failure).
```

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

If it’s possible to allow that in steams systems the maps there could also be used my non-steam players, though owning it on steam is still a lot easier to use. You need to know the ID of mindustry and the ID of the workshop item. One of the areas of interest is downloading maps remotely on a headless server, since you also need to enter steam guard code to log in which always requires interaction.

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by basxto** [comment]
Okay, I didn’t realize this has an extra repo, I only checked the issues of the other repo. This seems to be somewhat related to #29, but that doesn’t look like it was about anonymous login. It definitely works for me when I’m logged in, though I also own the game on steam.

**Comment by Everlost065** [comment]
I propose to introduce regenerators to the planet "Erekir" 
I understand that if you stand next to buildings it is possible to treat them, but it is not convenient to stand next to them and build a defense in parallel 🌊  

(I am writing through a translator 😅)

**Comment by Everlost065** [comment]
I propose to introduce regenerators to the planet "Erekir"
I understand that if you stand next to buildings it is possible to treat them, but it is not convenient to stand next to them and build a defense in parallel 🌊

(I am writing through a translator 😅)

**Comment by basxto** [comment]
> I am writing through a translator

You need to open a separate issue for this: https://github.com/Anuken/Mindustry-Suggestions/issues/new/choose

Reading something with a translator is okay. But when you write, it is better to additionally include the original text. Otherwise if the translator does something wrong nobody will understand what you intended to say.

I do not understand what you are trying to say, but I did not play Erekir yet. Do you talk about "rebuild" (build something up that was destroyed) or "repair" (restore health to not get it destroyed to begin with)?
